<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>955</width>
    <height>843</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>171</width>
      <height>551</height>
     </rect>
    </property>
    <property name="title">
     <string>Entrepot</string>
    </property>
    <widget class="QTreeView" name="treeView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>151</width>
       <height>511</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>30</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Wide Latin</family>
     </font>
    </property>
    <property name="text">
     <string>Filtre avanc√© : </string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxTypeModel">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>60</y>
      <width>181</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Tous</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Conteneur</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Palette</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Produit</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>120</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Id : </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>120</y>
      <width>311</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_Conteneur">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>160</y>
      <width>471</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>Caracteristics Conteneur</string>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Type : </string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_2">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>70</y>
       <width>271</width>
       <height>26</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>TOUS</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>NORMAL</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>FROID</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>FRAGILE</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>CapaciteMax : </string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>110</y>
       <width>271</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_18">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>30</y>
       <width>271</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_32">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Id : </string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_Palette">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>160</y>
      <width>471</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>Caracteristics Palette</string>
    </property>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Id : </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Destination : </string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_5">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>70</y>
       <width>271</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>DateEnvoiPrevue : </string>
     </property>
    </widget>
    <widget class="QDateEdit" name="dateEdit">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>110</y>
       <width>271</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_6">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>150</y>
       <width>271</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>CapaciteMax : </string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_19">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>30</y>
       <width>271</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_Produit">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>160</y>
      <width>471</width>
      <height>341</height>
     </rect>
    </property>
    <property name="title">
     <string>Caracteristics Produit</string>
    </property>
    <widget class="QLabel" name="label_25">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Type : </string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_9">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>80</y>
       <width>271</width>
       <height>26</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>TOUS</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ALIMENTAIRE</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ELECTRONIQUE</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>MEDICAMENT</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_26">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Nom : </string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_13">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>120</y>
       <width>271</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_27">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>DateEntreeStock : </string>
     </property>
    </widget>
    <widget class="QDateEdit" name="dateEdit_5">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>160</y>
       <width>271</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_14">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>200</y>
       <width>271</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_28">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>CapaciteMax : </string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_15">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>240</y>
       <width>271</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_29">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Poids : </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_30">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>280</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Volume : </string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_16">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>280</y>
       <width>271</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_17">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>40</y>
       <width>271</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_31">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Id : </string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>955</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCRUD">
    <property name="title">
     <string>CRUD</string>
    </property>
    <addaction name="actionAjoute_Produit"/>
    <addaction name="actionAjouter_Conteneur"/>
   </widget>
   <addaction name="menuCRUD"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAjoute_Produit">
   <property name="text">
    <string>Ajoute Produit</string>
   </property>
  </action>
  <action name="actionAjouter_Conteneur">
   <property name="text">
    <string>Ajouter Conteneur</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
